pcsx2/x86/ix86-32/iR5900-32.cpp

```diff
@@ -575,7 +575,6 @@ static void recAlloc()
 	recROM1 = basepos;
 	basepos += (Ps2MemSize::Rom1 / 4);
 	recROM2 = basepos;
-	basepos += (Ps2MemSize::Rom2 / 4);
```

Comment:
You might wanna check this with Tellow, I'm sure we had an arguement on if it should be there or not, I think he got me to put it back in because then you can see if the pointer is pointing to the right place (end of the data), but I forget.

Reply:
Yeah, not a fan

You could make a `BumpAllocator` struct that holds a pointer with a single method `allocate` that advances the pointer and returns the pre-advance value.  Then codacy won't care that it's modified without use at the end, because it doesn't care as long as you stick enough indirection in there